packages/react/src/components/Footer/__stories__/Footer.stories.js

```diff
@@ -54,13 +53,50 @@ Default.story = {
   parameters: {
     knobs: {
       Footer: ({ groupId }) => {
-        const languageOnly =
-          DDS_LANGUAGE_SELECTOR &&
-          boolean(
-            'switch to the language selector (languageOnly)',
+        /**
+         * Language callback demo function
+         *
+         * @param {string} selectedItem Selected item
+         */
+        const languageCallback = selectedItem => {
+          console.log(
+            'footer (language selector) selected item:',
+            selectedItem
+          );
+        };
```

Comment:
@kennylam caught a wild console log here

Reply:
@jeffchew I saw these – I assumed they were intentional to display the `lc-cc` when a selection is made and are part of the story. I can remove if we're not using anymore.

Reply:
Yeah I think removing would be fine.